user-export.sh

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
 
-echo "starting exporter"
+export ALL_USERS_FILE=${WORK_DIRECTORY}/EXPORT_SERVER_USERS.json
```

Comment:
why ALL_USERS_FILE and not just USERS_FILE?

Reply:
because,
in a step i get the entire user ids&usernames in a single get into the file ${WORK_DIRECTORY}/EXPORT_SERVER_USERS.json

then for each user i make separate get

the reason for that, is when you try to get all users, you do not get the entire related user information, but only the tope level ones (user Attributes cant be retrieved using "get users", instead "get users/$UID)
similar for groups (get users/$UID/groups)